servicex/resources/transformation/submit.py

```diff
@@ -191,10 +191,6 @@ def post(self):
                                                     routing_key=parsed_did.microservice_queue,
                                                     body=json.dumps(did_request))
             else:
-                # Request a preflight check on the first file
```

Comment:
This will break single file transform, since it will never start - I'll take a look at fixing this